# i don't exactly get whats going wrong here. but the build-system somehow
# calls the "ln" inside the "arch" dir, where a symlink pointing to itself is
# created... this will fool later stages of the build-process.
#
# anyhow, this seems to "fix" it -- meaning in the case observer, no symlink
# what-so-ever is created... meh...
#
# to be explcicit: this fix works accidentally, the root-cause was not understood ;-)
#
# after a bit of research: the commit 9a3fbdc18fca5adc117ad6b505a625859adb969e
# in git://github.com/hardkernel/linux.git removes the "arch-release" directory
# and 73341c84ade21b97091b0289a9ddfc00900c038b readds it.  besides this, this
# driver seems to be relatively calm... well, propriatary software development:
# one initial "eat-my-shit" dump, and off they go. stupid
--- linux/drivers/gpu/arm/mali400/ump/Kbuild	2014-07-30 15:44:03.984805492 +0200
+++ linux/drivers/gpu/arm/mali400/ump/Kbuild	2014-07-30 15:44:39.609139070 +0200
@@ -21,7 +21,7 @@
 else
 # Link arch to the selected arch-config directory
 $(shell [ -L $(src)/arch ] && rm $(src)/arch)
-$(shell ln -sf arch-$(CONFIG) $(src)/arch)
+$(shell ln -sf $(src)/arch-$(CONFIG) $(src)/arch)
 $(shell touch $(src)/arch/config.h)
 endif
 
